docs/products/kafka/howto/enable-kafka-tiered-storage.md

```diff
@@ -26,39 +18,47 @@ After activation, you must enable the feature from the
     page](https://aiven.io/pricing?product=kafka) for a comprehensive
     list of supported plans and regions.
 -   Aiven CLI
+-   The tiered storage for Aiven for Apache Kafka® is an **early availability**
+feature and requires activation on your Aiven account. Contact our sales team
+at sales@aiven.io to request activation. Once activated, you must
+enable tiered storage from the [feature preview](/docs/platform/howto/feature-preview)
+page in your user profile to start using it.
+
 
 ## Enable tiered storage via Aiven Console
 
 Follow these steps to enable tiered storage for your service using the
 Aiven Console.
 
-1.  Access the [Aiven console](https://console.aiven.io/), and select
+1. Log in to the [Aiven console](https://console.aiven.io/), and select
     your project.
-2.  Either create a new Aiven for Apache Kafka service or select an
+1. Either create a new Aiven for Apache Kafka service or select an
     existing one.
     -   For
         [a new service](/docs/platform/howto/create_new_service):
-        a.  On the **Create Apache Kafka® service** page, scroll down to
+
+        -  On the **Create Apache Kafka® service** page, scroll down to
             the **Tiered storage** section.
-        b.  Turn on the **Enable tiered storage** toggle to activate
+        -  Turn on the **Enable tiered storage** toggle to activate
```

Comment:
Shouldn't these sub-steps be numbered? They look like sequential steps.

Reply:
Changed them to sub-steps